### PR TITLE
integrated local-extract and aded to the dashboard

### PR DESCRIPTION
**Pull request by Elexy**

### CODE REVIEW

File: runner/config.js [hunk 1]
```diff
@@ -34,13 +34,18 @@ const getBenchmarkHostname = () => {
...
   {
-    name: 'unixFS',
-    shell: `rm -Rf /tmp/peerb && source ~/.nvm/nvm.sh && OUT_FOLDER=/tmp/out REMOTE=true node ${remoteTestsPath}/local-add.js`,
+    name: 'unixFS-add',
+    shell: `source ~/.nvm/nvm.sh && OUT_FOLDER=/tmp/out REMOTE=true node ${remoteTestsPath}/local-add.js`,
     localShell: 'OUT_FOLDER=/tmp/out REMOTE=true node ' + path.join(__dirname, '/../tests/local-add.js')
+  },
+  {
+    name: 'unixFS-extract',
+    shell: `source ~/.nvm/nvm.sh && OUT_FOLDER=/tmp/out REMOTE=true node ${remoteTestsPath}/local-extract.js`,
```
Comment (mcollina) [inline runner/config.js]: Can you extract the `source ~/.nvm/nvm.sh && OUT_FOLDER=/tmp/out REMOTE=true` part into a variable? So we don't repeat this everywhere with the risk of forgetting one.

File: runner/persistence.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ const writePoints = (data) => {
   for (let point of data) {
     payload.push({
       measurement: point.name,
-      tags: { subTest: point.subTest, commit: point.meta.commit, project: point.meta.project, testClass: point.testClass },
+      tags: { subTest: point.subtest, commit: point.meta.commit, project: point.meta.project, testClass: point.testClass },
```
Comment (mcollina) [inline runner/persistence.js]: why this change?

File: tests/local-add.js [hunk 3]
```diff
@@ -48,11 +48,11 @@ async function scenarios () {
...
     })
 
-    arrResults.push(await localAdd(node1, testName, 'add-empty-repo', 'smallfile'))
+    arrResults.push(await localAdd(node1, testName, 'empty-repo', 'smallfile'))
 
-    arrResults.push(await localAdd(node1, testName, 'add-populated-repo', 'smallfile'))
+    arrResults.push(await localAdd(node1, testName, 'populated-repo', 'smallfile'))
 
-    arrResults.push(await localAdd(node, testName, 'add-populated-repo', 'largefile'))
+    arrResults.push(await localAdd(node, testName, 'populated-repo', 'largefile'))
```
Comment (mcollina) [inline tests/local-add.js]: can you refactor this block by iterating over the list of tests (maybe from config)?

File: runner/persistence.js (hunk 2 above)
Comment (Elexy) [reply runner/persistence.js]: Ron used lowercase i used Snakecase. We should probably unify that everywhere ;-)

File: tests/local-add.js (hunk 3 above)
Comment (Elexy) [reply tests/local-add.js]: yep, I will make a new PR though
